core/src/main/java/nu/yona/server/messaging/entities/MessageDestination.java

```diff
@@ -5,21 +5,16 @@
 package nu.yona.server.messaging.entities;
```

Comment:
I understand caching all messages is too much of a leap. Please revert ``MessageDestination`` to the original design. We might need a discussion in front of a whiteboard to conclude on what change is needed here.